posix.mak

```diff
@@ -283,7 +283,7 @@ install :
 
 .PHONY : unittest
 ifeq (1,$(BUILD_WAS_SPECIFIED))
-unittest : $(addsuffix .d,$(addprefix unittest/,$(D_MODULES)))
+unittest : $(addsuffix .run,$(addprefix unittest/,$(D_MODULES)))
```

Comment:
this would be nice, too:

    .PHONY : $(addsuffix .run,$(addprefix unittest/,$(D_MODULES)))
    .PHONY : $(addsuffix .test,$(addprefix unittest/,$(D_MODULES)))